strapi/strapi
# enhancement: add support for codemods command to run on plugins

### What does it do?

Allows the codemods argument to run on plugins instead of only strapi projects

### Why is it needed?

To help plugin writers migrate their plugins to Strapi 5

### How to test it?

- run the upgrade tool `codemods` command against a plugin; it should work
- run it with `upgrade` and it should still require being in a Strapi project 
- run the above against a strapi project; it should still work

### Related issue(s)/PR(s)

Let us know if this is related to any issue/pull request
